frontends/election-manager/src/components/navigation_screen.tsx

```diff
@@ -111,7 +113,7 @@ export function NavigationScreen({
             {(isSystemAdministratorAuth(auth) ||
               isElectionManagerAuth(auth)) && (
               <React.Fragment>
-                <Button onPress={() => auth.logOut()} small>
+                <Button onPress={() => apiClient.logOut()} small>
```

Comment:
I know I'm not yet handling potential event handler errors as discussed during your presentation @jonahkagan, but I wanted to keep this PR from growing too much in scope

Reply:
Thanks for calling this out. I'm assuming you have a plan to revisit?

Reply:
Yes! Tracking miscellaneous cleanup items in https://github.com/votingworks/vxsuite/issues/2965